docs/content/develop/resource.md

```diff
@@ -210,6 +210,8 @@ For more information about types of resources and the generation process overall
    - If any of the added Go code (including any imports) is beta-only, change the file suffix to `.go.erb` and wrap the beta-only code in a version guard: `<% unless version = 'ga' -%>...<% else -%>...<% end -%>`.
 5. Register the resource in [`magic-modules/mmv1/third_party/terraform/utils/provider.go.erb`](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/utils/provider.go.erb) under "START handwritten resources"
    - Add a version guard for any beta-only resources.
+
+> **Note:** If you plan to use the [Add resource tests]({{< ref "/develop/test.md" >}}) guide to add your tests, make sure to keep the MMv1 configuration file.
```

Comment:
That flow makes sense, but I'm a little concerned about the user needing to jump around on multiple pages, sometimes needing to use the "Handwritten" tab, and sometimes using the "MMv1" tab. Would that be confusing?

I think you're right about deleting the YAML either way, but perhaps we could be a little more direct with just one additional step instead of my note:
```
6. Do not delete the MMv1 configuration file until it is no longer needed for other tasks.
    - [Add resource tests]({{< ref "/develop/test.md" >}})
    - [Add IAM support]({{<ref "#add-iam-support" >}})
```

Reply:
definitely a fan of avoiding jumping around where possible - and also a fan of reducing duplicated content 😂  I'm not sure offhand, maybe just do what you think is best for now?

Regarding simplifying the additional steps, my only concern is that having a step that is "do not do X" is maybe a little off-style-guide (but I don't see it explicitly called out). What would you think about something like:

```
6. Optional: Complete other handwritten tasks that require the MMv1 configuration file.
    - [Add resource tests]({{< ref "/develop/test.md" >}})
    - [Add IAM support]({{<ref "#add-iam-support" >}})
7. Delete the MMv1 configuration file.

Reply:
Agreed, I went with you last suggestion